src/Security/Authentication/JwtBearer/src/JwtBearerHandler.cs

```diff
@@ -288,7 +288,16 @@ protected override async Task HandleChallengeAsync(AuthenticationProperties prop
     protected override Task HandleForbiddenAsync(AuthenticationProperties properties)
     {
         var forbiddenContext = new ForbiddenContext(Context, Scheme, Options);
-        Response.StatusCode = 403;
+
+        if (Response.HasStarted)
```

Comment:
```suggestion
        if (Response.StatusCode == 403)
        {
            // No-op
        }
        else if (Response.HasStarted)
```

Reply:
Thank you for the feedback @Tratcher .  I implemented this suggested change when I pushed up the modification to the unit test.